src/Item/Item.model.ts

```diff
@@ -20,7 +20,7 @@ export class Item extends Model<ItemAttributes> {
         FROM ${raw(this.tableName)} items
         JOIN ${raw(
           ItemCuration.tableName
-        )} item_curations ON items.id = item_curations.item_id AND
+        )} item_curations ON items.id = item_curations.item_id
```

Comment:
I was thinking that the tree to be built requires not only this data, but the data of all the items that are published for a third party. We need all the data to re-build the tree, and the tree is built with all the items of a third party.

Reply:
True, I think then that the query should not check for curation status but rather for curation "existence", which is already being done by the INNER JOIN right?

Reply:
Alright, as Nico suggested, we can take care of that in another PR. Issue created: https://github.com/decentraland/builder-server/issues/439